pubsub/iam.js

```diff
@@ -84,14 +103,14 @@ function setTopicPolicyExample (topicName, callback) {
     ]
   };
 
-  // Retrieve the IAM policy for the provided topic
-  topic.iam.setPolicy(myPolicy, function (err, policy, apiResponse) {
+  // Set the IAM policy for the specified topic
+  topic.iam.setPolicy(myPolicy, function (err, policy) {
```

Comment:
Nit that can be ignored: I typically always avoid variable names like `mypolicy`. `newPolicy` or just `policy` would work fine.